annif/backend/yake.py

```diff
@@ -0,0 +1,196 @@
+"""Annif backend using Yake keyword extraction"""
+# TODO Mention GPLv3 license also here?
```

Comment:
Hmm...maybe it should be mentioned. At least we can't leave it as a TODO forever :)